.travis.yml

```diff
@@ -1,26 +1,27 @@
 language: python
-python: "2.7"
 sudo: false
 cache:
   directories:
   - eggs
-addons:
-  firefox: latest-esr
+  - downloads
+python:
+  - 2.7
+matrix:
+  fast_finish: true
 install:
-- python bootstrap-buildout.py
-- bin/buildout -c travis.cfg
+  - python buildout.py
```

Comment:
Why do you rename this? Is it possible to get rid of the file entirely and use ./bin/pip install -r requirements instead?

Reply:
`./bin/pip install -r requirements.txt` I tried this but the Travis CI messed up, not sure why so for now I keep just the default how the package came with it.